src/i18n/es.js

```diff
@@ -216,6 +216,31 @@ const messages_es = {
       transfer: 'Transferencia',
       valid_thru: 'válido hasta',
     },
+    purchase_summary: {
+      buy_button: 'COMPRAR',
+      discount_for_payment_paid: 'Saldo a favor:',
+      discount_for_prepayment: 'Descuento por pago adelantado:',
+      edit_add_recipients_button: 'Editar o agregar destinatarios',
+      error_message: 'Tu pago no pudo procesarse. Elige otro método de pago o inténtalo más tarde.',
+      explanatory_legend: 'La renovación es automática y puedes cancelarla cuando quieras. El precio del Plan no incluye impuestos.',
+      header: 'Resumen de compra',
+      month_with_plural: `{monthsCount, plural, one {# mes}other {# meses} }`,
+      months_to_pay: 'Meses a abonar:',
+      plan_standard_title: 'Plan STANDARD',
+      plan_type_monthly_deliveries: 'por Envíos',
+      plan_type_monthly_deliveries_label: 'Envíos:',
+      plan_type_prepaid: 'por Créditos',
+      plan_type_subscribers: 'por Contactos',
+      plan_type_subscribers_label: 'Contactos:',
+      promocode_label: 'Código de descuento',
+      promocode_legend: 'Ingresar un cupón de promoción',
```

Comment:
This is the same thing, shouldn't it be the same text?
```suggestion
      promocode_label: 'Código de Descuento',
      promocode_legend: 'Ingresar un Código de Descuento',
```
or 
```suggestion
      promocode_label: 'Cupón de Promoción',
      promocode_legend: 'Ingresar un Cupón de Promoción',
```
@jmonacchi what would be best?

Also the Promo Code is with capitals in Upgrade:
![image](https://user-images.githubusercontent.com/2439363/134902464-87b7cc9e-cd42-4b92-bc58-5ea37b8d9578.png)

Reply:
In Spanish I prefer Código de descuento instead of Código de Promoción. 
So promocode_label would be Código de descuento 
and promocode_legend would be Ingrésalo aquí (just like in Confluence)

What do you think @cbernat ?

Reply:
Done